kernel/src/engine/default/storage.rs

```diff
@@ -34,5 +34,5 @@ where
         .collect();
     let store = HdfsObjectStore::with_config(url.as_str(), options_map)?;
     let path = Path::parse(url.path())?;
-    return Ok((Box::new(store), path));
+    Ok((Box::new(store), path))
```

Comment:
do we have linting in ci to make committing this impossible in the first place?

Reply:
we do, we run rustfmt and clippy. surprised clippy didn't catch this..?

Reply:
Thanks for reminding me! It's because our CI didn't run clippy with `--all-features` so it wasn't checking this (it's behind the `cloud` feature.

I've updated the workflow file now to run with `--all-features`. Hopefully that's uncontroversial :)

Reply:
nice thanks :)